expat/Makefile.am

```diff
@@ -136,9 +136,12 @@ tests/xmlconf: tests/xmlts.zip
 run-xmltest: tests/xmlconf
 if WITH_XMLWF
 	$(MAKE) -C xmlwf
-	tests/xmltest.sh "$(PWD)/run.sh $(PWD)/xmlwf/xmlwf@EXEEXT@" 2>&1 | tee tests/xmltest.log
-	dos2unix tests/xmltest.log
-	diff -u tests/xmltest.log.expected tests/xmltest.log
+	export EXPAT_ABS_SRCDIR=$(abs_top_srcdir); \
+	export EXPAT_ABS_BUILDDIR=$(abs_top_builddir); \
+	$(srcdir)/tests/xmltest.sh "$(abs_top_builddir)/run.sh $(abs_top_builddir)/xmlwf/xmlwf@EXEEXT@" 2>&1 | \
+	tee $(builddir)/tests/xmltest.log
```

Comment:
It would be cool to have some extra indentation here to better support the line continuation.